io.openems.edge.controller.battery.batteryprotection/doc/description.txt

```diff
@@ -0,0 +1,6 @@
+UNDEFINED: 		Undefined status is always reached when one of the values are undefined 
+NORMAL: 		If no limit is reached this is the normal state
+LIMIT:			If voltage or temperature limits are reached, charging or discharging will be denied 
+FORCE_CHARGE:	If cell voltage or SoC is below a certain limit value, charging is forced
+FULL_CHARGE:	If battery is not charged or discharged for a certain time period a full charge has to be done
+CHECK:			If charging was forced, discharging is only allowed if SoC has increased
```

Comment:
This information could be added to readme.adoc file. Alternatively provide a ful State-Machine definition using mermaid.js - see https://github.com/OpenEMS/openems/blob/1353269b3765c16268030bace9a7ee97a866d75b/io.openems.edge.ess.generic/doc/statemachine.md

Ah - just saw you did it using drawio, that is also ok. Levaing this as a comment anyway.